tests/scripts/installandtest.sh

```diff
@@ -1,12 +1,18 @@
 #!/bin/bash
 
 set -x
-env | sort
+env | sort | tee ${ARTIFACT_DIR}/env.txt
 mkdir -p ~/.kube
 cp /tmp/kubeconfig ~/.kube/config 2> /dev/null || cp /var/run/secrets/ci.openshift.io/multi-stage/kubeconfig ~/.kube/config
 chmod 644 ~/.kube/config
 export KUBECONFIG=~/.kube/config
 
+if [ ! -d "${ARTIFACT_DIR}/screenshots" ]; then
+  ARTIFACT_SCREENSHOT_DIR="${ARTIFACT_DIR}/screenshots"
```

Comment:
Maybe move this var out of the `if` and use it in the condition?

Reply:
Good call. I created and exported `ARTIFACT_SCREENSHOT_DIR` so it is accessible by all tests running in this environment